staging/src/k8s.io/code-generator/cmd/validation-gen/output_tests/ordering/typedefs/testdata/validate-false.json

```diff
@@ -0,0 +1,23 @@
+{
+    "*typedefs.E00": {},
+    "*typedefs.E01": {
+      "": [
+        "E01, no flags"
+      ]
+    },
+    "*typedefs.E02": {
+      "": [
+        "E02, ShortCircuit"
```

Comment:
Currently I believe the test fixture looks at the error messages output by validateFalse for ShortCircuit (eg: the flag was passed).  Should it test that the validation short circuited, eg: it got a single error vs N errors in a case like that above?  I'm not sure if this is handled currently in the test fixture atm

Reply:
Thanks for checking these carefully!

The short circuiting is scoped to a validation node. In the case of `E02`, the validation is for:

```
// +validateFalse={"flags":["ShortCircuit"], "msg":"E02, ShortCircuit"}
type E02 string
```

So the scope is a single type with a single validation.

Note how E03 and Emultiple do have more validations defined on the types and how only the 1st ShortCircuit error is returned.